py/circuitpy_defns.mk

```diff
@@ -297,6 +300,9 @@ SRC_COMMON_HAL_ALL = \
 	busio/SPI.c \
 	busio/UART.c \
 	busio/__init__.c \
+	camera/__init__.c \
+	camera/Camera.c \
+	camera/ImageSize.c \
```

Comment:
Move this down to `SRC_BINDINGS_ENUMS` and you won't need the empty common-hal file.